css/style.css

```diff
@@ -0,0 +1,225 @@
+   left: 65px;
+}
+
+/* nav */
+.nav {
+    font-size: 18px;
+    letter-spacing: 0.02em;
+}
+
+.nav__link {
+    color: #e1c2f1;
+    font-weight: 700;
+    display: inline-block;
+    text-decoration: none;
+    margin: 0 25px 50px 0;
+}
+
+.nav__link:focus {
+    color: #fbf6e4;
+    border-bottom: 2px solid #FFEB3B;
+} 
+
+/* form */
+.form__control {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+        -ms-flex-direction: column;
+            flex-direction: column;
+    max-width: 260px;
+    margin-bottom: 20px;
+}
+
+.form__label{
+    color: #fbf6e4;
+    font-size: 16px;
+    font-weight: 700;
+    margin-bottom: 10px;
+    cursor: pointer;
+}
+
+.form__input {
+    height: 42px;
+    border: 2px solid rgba(191, 185, 204, 0.54);
+    background: none;
+    outline: none;
+    color: #fff;
+    font-size: 20px;
+    font-weight: 400;
+    -webkit-transition: background-color 0.3s ease-in-out;
+    -o-transition: background-color 0.3s ease-in-out;
+    transition: background-color 0.3s ease-in-out;
+}
+
+.form__input:focus {
+    border-color: #FFEB3B;
+}
+
+.form__input_password:focus:valid {
+    background-color: #f0e796;
+}
+
+.form__input_password:focus:invalid {
+    background-color: rgb(207, 58, 175);
+}
+
+/*  form checkbox */
+.form__label_checkbox {
+    color: #fbf6e4;
+    font-size: 13px;
+    font-weight: 700;
+}
+
+.form__input_checkbox[type='checkbox']{ 
+    height: 0; width: 0; 
+}
+
+.form__input_checkbox[type='checkbox'] + .form__label_checkbox{
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-align: center;
+        -ms-flex-align: center;
+            align-items: center;
+    -webkit-transition: color 250ms cubic-bezier(.4,.0,.23,1);
+    -o-transition: color 250ms cubic-bezier(.4,.0,.23,1);
+    transition: color 250ms cubic-bezier(.4,.0,.23,1);
+}
+
+span {
+    font-size: 0; 
+}
+
+.form__input_checkbox[type='checkbox'] + .form__label_checkbox > span{
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-pack: center;
+        -ms-flex-pack: center;
+            justify-content: center;
+    -webkit-box-align: center;
+        -ms-flex-align: center;
+            align-items: center;
+    margin-right: 10px;
+    width: 16px;
+    height: 16px;
+    background: transparent;
+    border: 2px solid #fff;
+    cursor: pointer; 
+    -webkit-transition: all 250ms cubic-bezier(.4,.0,.23,1); 
+    -o-transition: all 250ms cubic-bezier(.4,.0,.23,1); 
+    transition: all 250ms cubic-bezier(.4,.0,.23,1);
+}
+
+.form__input_checkbox[type='checkbox']:checked + .form__label_checkbox > span{
+    font-size: 13px; 
+    color: #FFEB3B;
+    border: 2px solid #FFEB3B;
+    -webkit-animation: shrink-bounce 200ms cubic-bezier(.4,.0,.23,1);
+            animation: shrink-bounce 200ms cubic-bezier(.4,.0,.23,1);
+}
+
+/* form button */
+
+button{
```

Comment:
Very unsafe to create element selectors. Assuming you have a lot of `button`'s in your HTML code they all will be affected by the current rule. Better to create a class